src/Ldap/LdapUserHydrator.php

```diff
@@ -63,6 +63,10 @@ public function hydrateUser(User $user, array $ldapEntry)
 
         $this->hydrateUserWithAttributesMap($user, $ldapEntry, $attributeMap);
 
+        if (is_array($user->getEmail())) {
+            $user->setEmail($user->getUsername()[0]);
```

Comment:
that doesn't look right

Reply:
So sorry Kevin, you are right, typo mistake. 
Changes requested sent.
Thanks a lot !